source/extensions/filters/common/rbac/matchers.h

```diff
@@ -221,6 +223,9 @@ class PolicyMatcher : public Matcher, NonCopyable {
   bool matches(const Network::Connection& connection, const Envoy::Http::RequestHeaderMap& headers,
                const StreamInfo::StreamInfo&) const override;
 
+  bool matches(const Network::Connection& connection, const Envoy::Http::RequestHeaderMap& headers,
```

Comment:
I notice that we have one `matches()` method which overrides a method and another `matches()` method which does not override a method. I think this might be slightly confusing. Can we rename the non-override method to something else? matchesCustom? customMatch? matchesWithCustomLibrary?